use-persisted-state.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module 'use-persisted-state' {
+  function createPersistedState<T>(key: string, storage: Storage): (initialState: T) => [S, Dispatch<SetStateAction<S>>];
```

Comment:
I had to make `storage` optional via `storage?: Storage`, since it has a default